sdk/sdktypes/value_scalar.go

```diff
@@ -151,10 +151,23 @@ func (BooleanValue) isConcreteValue() {}
 
 func (s BooleanValue) Value() bool { return s.read().V }
 
-func NewBooleanValue(v bool) Value {
+func mkbool(v bool) Value {
```

Comment:
`mkbool` is not a self-documenting function name. It's confusing even though it's private.

```suggestion
func boolValue(v bool) Value {
```

Reply:
done